ibmpisession/utils.go

```diff
@@ -96,3 +96,13 @@ func costructRegionFromZone(zone string) string {
 	reg, _ := regexp.Compile(regex)
 	return reg.ReplaceAllString(zone, "")
 }
+
+// SDKFailWithAPIError returns a custom error message if a HTTP error response 500 or greater is found
+func SDKFailWithAPIError(err error, errMessage string) error {
+	if apierr, ok := err.(*runtime.APIError); ok {
+		if apierr.Code >= 500 {
```

Comment:
Are you sure it works? I think the orchestration layer does not return errors eg: timeouts with json format. Hence the err object fails to unmarshal. Not sure how we can handle string responses from the API server.
IMO only way is to add the missing response code in the swagger file.